tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -338,6 +355,27 @@ private function get_mock_express_checkout_payment_method( string $payment_metho
+
+		$this->mock_gateway
+			->method( 'stripe_request' )
+			->willReturnCallback(
+				function ( $endpoint ) use ( $mock_payment_method ) {
+					if ( 0 === strpos( $endpoint, 'payment_methods/' ) ) {
+						return $mock_payment_method;
+					}
+					return (object) [];
+				}
+			);
+	}
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'mock_stripe_request_with_default_payment_method'. (undefined)

(UnusedPrivateMethod)